client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx

```diff
@@ -93,8 +123,14 @@ export const RequestLineEdit = ({
             sx={{ paddingLeft: 4, paddingRight: 4 }}
             justifyContent="space-between"
           >
-            <ConsumptionHistory id={draft?.id || ''} />
-            <StockEvolution id={draft?.id || ''} />
+            {draft?.isCreated ? (
+              <Box display="flex" height={289} />
```

Comment:
The height should probably just be removed. I was trying to help reduce the jankiness, and gave up. I think a little jank is ok as a trade off for having the charts show